int8 connected

float32 altitude
float32 longitude
float32 latitude

float32 speedKnots

int32 numberOfSatellites

int64 timeStamp
